### DISCUSSION TITLE
IMX500 Sony AI Accelerator Ultralytics model export = imx Convertion error

### DISCUSSION THREAD

**Discussion by ARCHANGELOVE** [community/community#145525] [Programming Help]
### Select Topic Area

Question

### Body

Hi everyone im facing error on convertion YOLOv8n-obb model to imx. Is this a bug

this is the command i run 

 yolo export model=yolov8n-obb.pt format=imx

  File "/home/ubuntu/.pyenv/versions/3.9.20/lib/python3.9/site-packages/ultralytics/nn/modules/head.py", line 124, in _inference
    dbox = self.decode_bboxes(
TypeError: decode_bboxes() got an unexpected keyword argument 'xywh'

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @ARCHANGELOVE!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
